index.html

```diff
@@ -2019,7 +2029,7 @@ <h2>The <dfn>ExposedThing</dfn> interface</h2>
     </p>
     <pre class="idl">
       [SecureContext, Exposed=(Window,Worker)]
-      interface ExposedThing: ConsumedThing {
+      interface ExposedThing {
```

Comment:
I wonder whether it makes sense to create an *abstract* interface Thing

```
interface Thing {
  constructor(ThingDescription td);
  ThingDescription getThingDescription();
}
```

which gets extends by `ConsumedThing `and `ExposedThing`.

I believe doing so avoids duplicating the internal slots in both interfaces.

Reply:
They are _separate_ objects with separate internal slots. Internal slots are not inherited. 
On the other hand, this practice is discouraged in Web APIs.

Reply:
As said, I don't have a strong opinion and it is fine like it is now.

Having said that, I am pretty sure that if it were me to implement it I would probably share the code ;-)

Reply:
ExposedThing implementations are free to do whatever that makes sense, including extending the object.